### DISCUSSION TITLE
生态系统 | FixIt

### DISCUSSION THREAD

**Discussion by giscus** [hugo-fixit/FixIt#432] [General]
# 生态系统 | FixIt

Hugo FixIt 生态系统，包括：文档、主题组件、开发工具、第三方组件等。

FixIt 主题旨在在保持 **简洁** 的同时，提供强大的扩展能力。为此，我们开发了一系列额外的 Hugo 主题组件供用户选择使用。

https://fixit.lruihao.cn/zh-cn/components/

**Comment by Lruihao** [comment]
如果你开发了相关的主题组件，可以分享出来哦～😄
